service/main.ml

```diff
@@ -118,8 +118,11 @@ let main config mode app capnp_address github_auth submission_uri =
   add_default_matching_log_rules ();
   Lwt_main.run begin
     let listen_address = Capnp_rpc_unix.Network.Location.tcp ~host:"0.0.0.0" ~port:Conf.Capnp.internal_port in
-    Capnp_setup.run ~listen_address capnp_address >>= fun (vat, rpc_engine_resolver) ->
-    let ocluster = Capnp_rpc_unix.Vat.import_exn vat submission_uri in
+    Capnp_setup.run ~listen_address capnp_address >>= fun (_vat, rpc_engine_resolver) ->
+    let ocluster =
+      let vat = Capnp_rpc_unix.client_only_vat () in
+      Capnp_rpc_unix.Vat.import_exn vat submission_uri
```

Comment:
:warning: I have no idea why it didn't work with the previous code for the `vat` and what impact this change will have on production, I need to check what's going on.

Reply:
With the previous code, I'm getting:
```
current_ocluster [WARNING] Error connecting to build cluster (will retry):
Failure("Failed: Unknown persistent service ID")
```

Reply:
Are you using the same private key for both processes? If so, it will think you want it to connect to itself.